<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="textNumberLable">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchNumber">
      <property name="text">
       <string>Найти номер</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>391</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="number_4">
        <property name="text">
         <string>        ФИО        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fioLable"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="number">
        <property name="text">
         <string>        Номер        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberLable"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="number_6">
        <property name="text">
         <string>Номер парковочного места</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberParkingSpaiceLable"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="number_3">
        <property name="text">
         <string>Время остановки на парковке</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timeStopLable"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="number_8">
        <property name="text">
         <string>Другая информация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="otherInformationLable"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="number_7">
        <property name="text">
         <string>В черном ли листе</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blackListLable"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;При удалении номера:&lt;/p&gt;&lt;p&gt;- найти номер&lt;/p&gt;&lt;p&gt;- достовериться, что нашелся тот номер&lt;/p&gt;&lt;p&gt;- нажать кнопку &amp;quot;Удалить кнопку&amp;quot;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;При сохранении нового номера:&lt;/p&gt;&lt;p&gt;- обязательно внести Номер, ФИО, Дату покупки места, Время остановки на парковке.&lt;/p&gt;&lt;p&gt;Номер парковочного места, Находиться ли номер в черном списке и другую информацию&lt;/p&gt;&lt;p&gt;указывать не обязательно &lt;/p&gt;&lt;p&gt;- нажать кнопку &amp;quot;Сохранить номер&amp;quot;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;При изменении информации номера:&lt;/p&gt;&lt;p&gt;- найти номер &lt;/p&gt;&lt;p&gt;- изменить информацию (остальное оставить как есть)&lt;/p&gt;&lt;p&gt;- нажать кнопку &amp;quot;Сохранить изменения&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>540</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="deleteNumber">
      <property name="text">
       <string>Удалить номер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveNumber">
      <property name="text">
       <string>Сохранить номер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveInform">
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openBDNumber">
      <property name="text">
       <string>Открыть базу данных номеров</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
